R/dqa4mortality/README.Rmd

```diff
@@ -0,0 +1,169 @@
+and was constructed to ensure accurate and timely data availability (NCHS 2024). Death certificate
+data, however, have at times been found to incomplete, inaccurate, and untimely, affecting the
+utility of the death certificate data
+([Flagg 2021](https://stacks.cdc.gov/view/cdc/100414)).
+To address some of these challenges, over the
+last decade, 54 of 57 jurisdictions implemented an electronic death reporting system (EDRS). These
+EDRSs allow for electronic capture of data, thus gaining efficiency in data sharing and reporting.
+
+Although the use of EDRSs, especially with embedded edit check tools (e.g.,
+[Validations and Interactive Edits Web Service](https://www.cdc.gov/nchs/data/nvss/modernization/VIEWS-Technical-User-Info-508.pdf)),
+has improved data quality,
+[challenges persist (Flagg 2021)](https://stacks.cdc.gov/view/cdc/100414).
+Comprehensive, monitoring of data quality over time is essential to identify gaps and identify
+opportunities to improve data quality.
+
+dqa4mortality implements functions to execute Data Quality Assessment
+Framework fort mortality data.
+
+All code in this section of the repository is intended for R users. If you prefer to use Python, please see [python/dqa4mortality](python/dqa4mortality).
+
+## Installation
+
+You can install dqa4mortality from [GitHub](https://github.com/) with:
+
+``` r
+# install.packages("devtools")
+devtools::install_github("nightingaleproject/Mortality-Data-Quality-Assessment-Examples/R/dqa4mortality")
+```
+
+## Data Requirements
+
+Data used for metric calculation should have a format where each row corresponds to one death record and each column corresponds to an attribute for the death record, such as "Date Certified" or "Underlying Cause of Death". ICD codes included in the data should not include periods (".").
+
+You can load your CSV data by using the following function, replacing "file_path" with the path to your mortality CSV:
+```{r, eval=FALSE}
+library(dqa4mortality)
+death_records <- load_death_records(file_path)
+```
+
+
+Example data is included in this package, called *synthetic_death_records*. To look at the first few rows and columns of this example data:
+
+```{r}
+library(dqa4mortality)
+head(synthetic_death_records[,1:33])
+```
+
+
+The raw CSV is also included in this repository; please see [SyntheticDeathRecordData.csv](data/SyntheticDeathRecordData.csv).
+
+## Functions
+
+dqa4mortality includes several functions to assess quality of your given mortality data. Functions beginning with "proportion\*" calculate metrics across the whole dataset, whereas those beginning with "certifier_proportion\*" calculate metrics by certifier within the dataset.
+
+Functions to assess data quality are:
+
+  * *(proportion/certifier_proportion)_not_certified_within_required_period()* – Proportion of records that were not certified within the expected time – jurisdictions typically specify how quickly a death should be certified after the actual date of death.
```

Comment:
The italics in these bullets don't seem to be coming through on GitHub, e.g., look at https://github.com/nightingaleproject/Mortality-Data-Quality-Assessment-Examples/blob/r-package/R/dqa4mortality/README.md#functions